packages/snap-preact/src/Snap.tsx

```diff
@@ -265,6 +265,65 @@ export class Snap {
 	};
 
 	public handlers = {
```

Comment:
Maybe we could put the different handlers into a separate file (similar to the events) to keep things tidy?

Reply:
The handlers require the tracker via `this.tracker` in the file so we'd be passing things around. Didn't make this change